atc/db/job.go

```diff
@@ -51,9 +51,12 @@ type Job interface {
 	GetNextPendingBuildBySerialGroup(serialGroups []string) (Build, bool, error)
 
 	ClearTaskCache(string, string) (int64, error)
+
+	SetNewInputs(bool) error
+	NewInputs() bool
```

Comment:
nit: Could this be called `HasNewInputs`/`SetHasNewInputs` instead? `NewInputs` sounds like it would return the inputs. (This goes for the column name too.)

Reply:
Sure thing, Ill fix it tomorrow.